modules/host/negotiateformcontract.go

```diff
@@ -128,6 +129,7 @@ func (h *Host) managedRPCFormContract(conn net.Conn) error {
 	// Send the host settings to the renter.
 	err := h.managedRPCSettings(conn)
 	if err != nil {
+		h.log.Debugln("RPCSettings call failed during form contract:", err)
 		return err
```

Comment:
I've found that if you add enough context, it's easy to grep for the exact error. For example, the `proto` error code looks like this:

```go
if err := encoding.WriteObject(conn, rev); err != nil {
	return errors.New("couldn't send revision: " + err.Error())
}
if err := modules.ReadNegotiationAcceptance(conn); err != nil {
	return errors.New("host did not accept revision: " + err.Error())
}
if err := encoding.WriteObject(conn, signedTxn.TransactionSignatures[0]); err != nil {
	return errors.New("couldn't send transaction signature: " + err.Error())
}
```

The only downside here is that you can't check for sentinel errors. For that, we'd want to use something like `pkg/errors` to wrap and unwrap the underlying error in context.